oaknational/Oak-Web-Application
# fix: update curric units params-incorrect error to 404

## Description

- Added 404 to params incorrect error

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1889--oak-web-application.netlify.thenational.academy
2. Click on /beta/teachers-2023/curriculum/NOT_A_VALID_PATH/units
3. You should see a 404 erro (rather than a 500 error)
